.tekton/marin3r-operator-image-pull-request.yaml

```diff
@@ -29,6 +39,12 @@ spec:
     value: 5d
   - name: dockerfile
     value: Containerfile.marin3r-operator
+  - name: prefetch-input
+    value: '{"type": "gomod", "path": "marin3r/api/"}'
```

Comment:
so now, we are just prefetching api/go.mod, what about the base ./go.mod for operator? Marin3r go mod has been split to api and root to reduce the dependency link when importing marin3r APIs for example, in RHOAM.

Reply:
updated, changed to   
- name: prefetch-input
    value: '[{"type": "gomod", "path": "marin3r/"}, {"type": "gomod", "path": "marin3r/api/"}]'
Not sure if it will work, as maybe need split to two tasks . It will be more complex